doc/sphinx/changes.rst

```diff
@@ -331,13 +331,11 @@ Commands and options
   (`#15760 <https://github.com/coq/coq/pull/15760>`_,
   by Pierre Roux).
 - **Added:**
-  A deprecation warning that :n:`>` in a field of a :cmd:`Class`
-  and the :g:`Class >` syntax will in the future also
-  declare :ref:`coercions <coercions>` as they do when used in
+  A deprecation warning that the :g:`Class >` syntax, that is currently
```

Comment:
Shouldn't this entry be fully removed?

Reply:
There were two deprecations. One for the `Class >` syntax and the other for the `:>` syntax in the field of a class. The latter will be delayed. The former actually does nothing and is dead code, so is being deprecated now and then removed.

Reply:
OK, I see, thanks.